### DISCUSSION TITLE
add file to release

### DISCUSSION THREAD

**Discussion by Kijan** [release-drafter/release-drafter#1200] [Q&A]
hi
i am new to github and github actions.
got the release drafter to work. took me a while...
but one thing i did not get working and dont know if it is possible...
i tested another release action and there i was able to add an file to the release. So someone who is interested could only download this file without the whole project.
can i do this with the release-drafter also? its a file in the main folder.

**Comment by jetersen** [comment]
You simply combine actions.

Note the action outputs: https://github.com/release-drafter/release-drafter#action-outputs

See how other people use it: https://github.com/search?q=action-gh-release+release-drafter&type=code

They have combined action-gh-release and release-drafter

**Reply by Kijan** [reply]
i dont get it...
i tried a few things but nothing works...
on one hand i get 
`GitHub Releases requires a tag`
on the other when i add 
`if: startsWith(github.ref, 'refs/tags/')`
then it will not upload anything...
the release has tag but i dont know how to give it to action-gh-release.
if i use 
`upload_url: ${{ steps.update_release_draft.outputs.upload_url }}`
like in one example i get 
`Unexpected input(s)`
i tried with
`tag_name: ${{ steps.update_release_draft.outputs.tag_name }}`
but did not work either

**Reply by jetersen** [reply]
Again see the usages:
https://github.com/jumpserver/koko/blob/394330916783903986ab65ef02d327391156c906/.github/workflows/release-drafter.yml

They run their workflow on tags, note the on section.
